src/util/config/index.js

```diff
@@ -5,9 +5,12 @@ import path from 'path';
 import chalk from 'chalk';
 import convict from 'convict';
 
-import buildConfig, { schema as buildSchema } from './build';
+import { env } from 'mwp-cli/src/config';
```

Comment:
my previous comment was based on looking at meetup-web-platform master.

I'm not 100%, but I suspect that similar techniques might help eliminate this dependency as well. I doubt asset_server.host, asset_server.port, asset_server.protocol are needed by meetup-web-platform, etc, i'm sorting through to figure out what is actually used.